sound/soc/sof/sof-priv.h

```diff
@@ -54,6 +54,9 @@ extern int sof_core_debug;
 	(IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_ENABLE_DEBUGFS_CACHE) || \
 	 IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST))
 
+/* So far the master core on all DSPs has ID 0 */
+#define SOF_DSP_MASTER_CORE 0
```

Comment:
its not possible to get this from platform files or something?

Reply:
@abonislawski I don't see a good place for this ATM. On Intel / HDA platforms we could add this to `struct sof_intel_dsp_desc` but it isn't used by other arches (iMX8). For now this is a place-holder which is at least marking the potential need for making this more flexible instead of assuming the value implicitly.